apps/aeutils/src/aeu_env.erl

```diff
@@ -7,13 +7,48 @@
 %%%-------------------------------------------------------------------
 -module(aeu_env).
 
--export([get_env/3]).
+-export([get_env/2, get_env/3]).
 
+-export([user_config/0, user_config/1]).
```

Comment:
I see that `get_env/{2,3}` and `user_config/1` are two separate sets of configs. Ok. Comment (not in this PR) would help.

Reply:
yes, the config data from the user config files is stored as a "private" environment variable, partly to simplify debugging (you can easily inspect the structure to see that it conforms to expectation), and partly to also support efficient dynamic checking (such as e.g. in `aehttp_app`. The `user_config/[0,1]` functions are meant to keep the name of the "private namespace" from leaking out.